README.md

```diff
@@ -1994,6 +1995,9 @@ notify, say, one or more of your devices as well as one for your spouse. As you
 can see in the example, you can even specify an optional sound to be played for
 the individual users. For a list of available sounds see the [Pushover API List](https://pushover.net/api#sounds).
 
+You can also specify the devices that should be notified, this is a comma-separated list.
```

Comment:
please add "comma-separated list of device names specified as a single string" to ensure people don't try to add a Python-like array or something.

Reply:
Sure, more clarification is always good